### DISCUSSION TITLE
There is no Vercel account linked with this GitHub account and lost all projects and paid domain name

### DISCUSSION THREAD

**Discussion by zneib** [vercel/community#1716] [Help] [answered]
### Summary

Lately I was receiving the message that "There is no Vercel account linked with this GitHub account" when attempting to login. I've always logged in through GitHub without any issue previously.

Today I tried logging in through the primary email that is associated with the my GitHub account and I was able to successfully login but, all of my projects, paid domains, etc. everything is missing!

Any help here is greatly appreciated!

**Comment by amyegan** [comment, marked as answer]
Hi @zneib. Please contact Vercel Support by emailing [registration@vercel.com](mailto:registration@vercel.com). Make sure you let them know the email address and GitHub username associated with your account so they can look up relevant logs and records to help you 🙏

**Reply by zneib** [reply]
Thank you!
